### ISSUE TITLE
Error compile

### ISSUE THREAD

**Issue by dieg013** [hansemannn/titanium-firebase-auth#2]
Hello, when I try to compile my application I have this error can you help me thank you
```
[TRACE] Undefined symbols for architecture i386:
[TRACE]  "_OBJC_CLASS_$_SFSafariViewController", referenced from:
[TRACE]  objc-class-ref in FirebaseAuth(FIRAuthURLPresenter_520787b8b17e6d9186ab9bac456f20c5.o)
[TRACE] ld: symbol(s) not found for architecture i386
[TRACE] clang: error: linker command failed with exit code 1 (use -v to see invocation)
[ERROR] ** BUILD FAILED **
```

**Comment by hansemannn** [comment]
Mhh, interesting. Then it needs the `SafariServices` to be added. Will fix now.

**Comment by hansemannn** [comment]
Fixed as part of 1.0.1 ~ https://github.com/hansemannn/titanium-firebase-auth/releases/tag/ios-1.0.1

**Comment by dieg013** [comment]
Thank ;)
